app/models/student_identifier_type.rb

```diff
@@ -0,0 +1,10 @@
+# frozen_string_literal: true
+class StudentIdentifierType < ActiveRecord::Base
+  default_scope { where(deleted_at: nil) }
+
+  belongs_to :organization, -> { unscope(where: :deleted_at) }
+
+  enum content_type: [:text, :number, :email]
```

Comment:
Hi @jasonlong, thanks for the feedback!

What do we think about using a `flash` to alert the teacher?

![screen shot 2016-06-10 at 4 40 32 pm](https://cloud.githubusercontent.com/assets/1311594/15978098/33173800-2f2a-11e6-8305-5058dd7354f3.png)

Also, to implement the identifier feature we will need to display student identifiers on the group assignment page. So I'm proposing to modify the group assignment page so that we have some space left to show other information about the student including student identifiers:

**Before:**

![screen shot 2016-06-10 at 4 44 00 pm](https://cloud.githubusercontent.com/assets/1311594/15978254/f91cb426-2f2a-11e6-9a33-db7587f1e920.png)

**After:**

![screen shot 2016-06-10 at 4 39 45 pm](https://cloud.githubusercontent.com/assets/1311594/15978267/005a612a-2f2b-11e6-8e98-bd6fede85e35.png)

In the modified version we can see that student identifiers were placed on the right of student login.

Haven't updated the code but it would be great if you can give me some feedback on this. Thanks!